src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/request.html.twig

```diff
@@ -187,6 +177,32 @@
             </div>
         </div>
 
+        <div class="tab {{ collector.requestcookies.all is empty and collector.responsecookies.all is empty ? 'disabled' }}">
+            <h3 class="tab-title">Cookies</h3>
+
+            <div class="tab-content">
+                <h3>Cookies Received</h3>
```

Comment:
No strong opinion.. i think i've tried to avoid those terms because of the `Request | Response` tabs already on the left (which also shows the cookies as raw headers). This is more the makeup version :)